pkg/util/mount/nsenter_mount.go

```diff
@@ -337,6 +337,13 @@ func (mounter *NsenterMounter) SafeMakeDir(subdir string, base string, perm os.F
 }
 
 func (mounter *NsenterMounter) GetMountRefs(pathname string) ([]string, error) {
+	exists, err := mounter.ExistsPath(pathname)
```

Comment:
Can we use a util method e.g. `PathExists(path string)` in `pkg/volume/util/util.go` instead?